bin/templates/cordova/lib/build.js

```diff
@@ -409,7 +457,31 @@ function parseOpts(options, resolvedTarget) {
             console.warn('Build option \'' + options[i] + '\' not recognized (ignoring).');
         }
     }
+    if (packageArgs.keystore && packageArgs.alias) {
+        var keystore = path.relative(ROOT, path.resolve(packageArgs.keystore));
+        ret.packageInfo = new PackageInfo(keystore, packageArgs.alias, packageArgs.keystorePassword,
+            packageArgs.password, packageArgs.keystoreType);
+    } else if (fs.existsSync(buildConfig)) {
```

Comment:
Should just check: `if (buildConfig)`, and show an error if it's set and the file doesn't exist.